engines/qemu/test-image/setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -e
+
+# Install taskcluster-worker for the qemu-guest-tools
+sudo cp /mnt/sr0/taskcluster-worker /opt/
+sudo chmod +x /opt/taskcluster-worker
+
+# Install a startup script to launch taskcluster-worker after X.org
+cp /mnt/sr0/worker.sh /home/tc/.X.d/worker.sh
+chmod +x /home/tc/.X.d/worker.sh
```

Comment:
These look like hard coded paths that might not exist on some people's machines - does this need to be moved out into a configuration file / platform-specific file?